<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #D0D0D0;</string>
  </property>
  <widget class="QLabel" name="exam_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Montserrat';
font-style: normal;
font-weight: 700;
font-size: 25px;
line-height: 122px;
color: #000;</string>
   </property>
   <property name="text">
    <string>ФИО студента</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_main_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font-family: 'Montserrat';
font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 16px;
color: #000;
border: none;
text-align:center;
}
QPushButton:hover {
color:#fff
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="mark_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Montserrat';
font-style: normal;
font-weight: 500;
font-size: 25px;
line-height: 122px;
color: #000;</string>
   </property>
   <property name="text">
    <string>Оценка</string>
   </property>
  </widget>
  <widget class="QLabel" name="mark_field">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Montserrat';
font-style: normal;
font-weight: 700;
font-size: 25px;
color: #000;</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_exam_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 background-color: #62B1FF;
	border-radius: 20%;
	font-family: 'Montserrat';
	font-style: normal;
	font-weight: 400;
	font-size: 15px;
	color: #000000;
}
QPushButton:hover{
   background-color:#7DBFFF;
}



</string>
   </property>
   <property name="text">
    <string>Удалить экзамен</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_mark_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 background-color: #62B1FF;
	border-radius: 20%;
	font-family: 'Montserrat';
	font-style: normal;
	font-weight: 400;
	font-size: 15px;
	color: #000000;
}
QPushButton:hover{
   background-color:#7DBFFF;
}



</string>
   </property>
   <property name="text">
    <string>Изменить оценку</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
